bugbug/bug_features.py

```diff
@@ -481,6 +481,22 @@ def __call__(self, bugs, **kwargs):
         return bugs[0]["target_milestone"] == bugs[1]["target_milestone"]
 
 
+class is_first_affected_same(couple_bug_feature):
+    def __call__(self, bugs, **kwargs):
+        version_1 = [
+            re.sub("cf_status_firefox_esr|cf_status_firefox", "", change)[:2]
```

Comment:
The field can have different values, `affected` is just one of them. This will not work in many cases, e.g. if bug 1 has cf_status_firefox51=unaffected and cf_status_firefox52=affected and bug 2 has cf_status_firefox52=affected, this will return false, but instead it should be true (the first affected version is 52).